mapstructure/mapstructure_examples_test.go

```diff
@@ -238,7 +238,7 @@ func ExampleDecodeSlicePath() {
 	// []mapstructure.NameDoc{mapstructure.NameDoc{Name:"bill"}, mapstructure.NameDoc{Name:"lisa"}}
 }
 
-func ExampleDecodeWithEmbeddedSlice() {
```

Comment:
What is the motivation behind changing these func names? Something to do with `Example`s?